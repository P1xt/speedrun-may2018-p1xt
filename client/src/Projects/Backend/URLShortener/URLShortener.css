.URLShortener {
  text-align: left;
  margin-top: 40px; }
  .URLShortener > h1 {
    text-transform: none; }
