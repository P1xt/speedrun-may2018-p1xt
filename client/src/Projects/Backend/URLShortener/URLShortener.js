import React from "react";
import { Container, Row, Col } from 'reactstrap';

import "./URLShortener.css";

const URLShortener = () => {
  return (
    <div className="URLShortener">
      <Container>
        <Row>
          <Col>
            <h1>U R L Shortener</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default URLShortener;
