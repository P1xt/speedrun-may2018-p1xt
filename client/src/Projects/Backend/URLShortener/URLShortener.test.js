import React from "react";
import renderer from 'react-test-renderer';
import { shallow } from "enzyme";

import URLShortener from "./URLShortener";

describe("<URLShortener />", () => {
  it("renders without crashing", () => {
    shallow(<URLShortener />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<URLShortener />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
