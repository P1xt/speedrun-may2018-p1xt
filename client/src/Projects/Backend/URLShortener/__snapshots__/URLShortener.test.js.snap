// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<URLShortener /> renders correctly 1`] = `
<div
  className="URLShortener"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          U R L Shortener
        </h1>
      </div>
    </div>
  </div>
</div>
`;
